Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -336,13 +337,17 @@ private extension HTTPClient {
 
         Logger.debug(Strings.network.api_request_started(request.httpRequest))
 
-        let task = session.dataTask(with: urlRequest) { (data, urlResponse, error) -> Void in
+        let task = self.session.dataTask(with: urlRequest) { (data, urlResponse, error) -> Void in
             self.handle(urlResponse: urlResponse,
                         request: request,
                         urlRequest: urlRequest,
                         data: data,
                         error: error)
         }
+
+        if #available(iOS 15.0, *) {
+            task.delegate = RedirectLoggerTaskDelegate()
+        }
```

Comment:
we should always log - the load shedder happens automatically and isn't dependent on OS version. The only thing that is dependent on OS version is Offline Entitlements

Reply:
`URLSessionTask.delegate` is iOS 15 only, but I'll do this using `URLSession.delegate` instead.

Reply:
Oh wait, but that delegate doesn't have anything for following redirects...